.storybook/themes/news.scss

```diff
@@ -2363,55 +2366,61 @@
 						"input-input": (
 							"font-family": var(--font-family-primary),
 							"font-weight": var(--global-font-weight-7),
-							"padding-inline-start": var(--global-spacing-7),
-							"width": 100%
+							"inline-size": 100%,
+							"padding-inline-start": var(--global-spacing-7)
 						)
 					)
 				),
 				"search-results-list-results": (
 					"components": (
 						"button": (
-							"margin": var(--global-spacing-2) auto 0,
-							"width": auto
+							"inline-size": auto,
+							"margin-block-end": 0,
+							"margin-block-start": var(--global-spacing-2),
+							"margin-inline-end": auto,
+							"margin-inline-start": auto
 						)
 					)
 				),
 				"search-results-list-results-item": (
 					"components": (
 						"attribution": (
 							"display": block,
-							"margin-top": var(--global-spacing-4)
+							"margin-block-start": var(--global-spacing-4)
 						),
 						"heading": (
 							"font-size": var(--heading-level-5-font-size),
 							"font-weight": var(--heading-level-5-font-weight),
-							"line-height": var(--heading-level-5-line-height),
-							"margin-bottom": var(--global-spacing-4)
+							"line-block-size": var(--heading-level-5-line-height),
```

Comment:
@malavikakoppula , `line-block-size` is still showing in the storybook `news.scss`, could you regenerate those styles?